libqtile/core/manager.py

```diff
@@ -1224,6 +1224,49 @@ def cmd_spawn(self, cmd):
             os.close(r)
             return int(pid)
 
+    def cmd_spawn_terminal(self, cmd=[]):
+        """Try to guess terminal or read default from config and open empty or
+        spawn command
+
+        cmd may be a string, which is parsed by shlex.split, or a list (similar
+        to subprocess.Popen).
+        """
+        if isinstance(cmd, str):
+            args = shlex.split(cmd)
+        else:
+            args = list(cmd)
+
+        TEST_TERMINALS = [
+                ['x-terminal-emulator', '-?'],
+                ['xterm', '-help'],
+                ['urxvt', '-help'],
+                ['st', '-v'],
```

Comment:
Let's add a few more:
* gnome-terminal
* terminator
* lxterminal
* mate-terminal
* roxterm
* tilda
* guake
* eterm
* sakura
* konsole
* hyper
* alacritty

Reply:
I don't think we need so much because we need basic terminal what should be installed by default. But ok - this don't cost much

Reply:
Well if you come from a GNOME environment, i.e. Ubuntu, I think that you might only have gnome-terminal installed. I do agree though that we could remove the terminals that doesn't come by default with any known distribution, à la alacritty or hyper.

Reply:
all debian based distributions has `x-terminal-emulator`

Reply:
Alright, Ubuntu was a bad example, but there are other distributions than Debian-based distributions. :stuck_out_tongue: